src/Microsoft.Tye.Core/GenerateKubernetesManifestStep.cs

```diff
@@ -12,6 +12,8 @@ public sealed class GenerateKubernetesManifestStep : ServiceExecutor.Step
 
         public string Environment { get; set; } = "production";
 
+        public string Namespace { get; set; } = "default";
```

Comment:
I think we need to be able to specify the namespace in the `tye.yaml` file as well.

```yaml
namespace: mynamespace
name: myapp
...
```

Potentially, the namespace information could be used by `tye run` as well for isolation.

Reply:
Yeah, it would be nice to have if this could be read from the config as well. I can see the reason why you'd want both - we don't currently expose the concept of environments to the user, so it makes sense to give manual control via CLI - and to make it possible to check in.

Reply:
done.